### ISSUE TITLE
After update i can't use Multiple-input in dialog

### ISSUE THREAD

**Issue by andjelko** [unclead/yii2-multiple-input#128]
I used Multiple-input in dialog ver. 1.3.1. and everything work fine, after update I can't add new row?
button doesn't work

**Comment by unclead** [comment]
how to reproduce a bug? need an example of code

**Comment by andjelko** [comment]
also i have maskinput in one field and when i add row (work when i am not in dialog) I get this error

VM596:1 Uncaught ReferenceError: inputmask_f27f15a7 is not defined(…)(anonymous function) @ VM596:1
r @ jquery.multipleInput.min.js:1
(anonymous function) @ jquery.multipleInput.min.js:1
dispatch @ jquery.js:4737
elemData.handle @ jquery.js:4549

**Comment by andjelko** [comment]
I use this extension for create dialog forms johnitvn/yii2-ajaxcrud
and your example for multipleinput

**Comment by unclead** [comment]
show me your code to reproduce this bug

**Comment by andjelko** [comment]
form
```
<?php
use yii\helpers\Html;
use kartik\form\ActiveForm;
use unclead\widgets\MultipleInput;
use yii\widgets\MaskedInput;

?>

<div class="form">

    <?php $form = ActiveForm::begin(['type'=>ActiveForm::TYPE_VERTICAL, 'id'=>'form' ]); ?>

    <?= $form->field($model, 'rowmodel')->widget(MultipleInput::className(), [
        'limit'             => 20,
        'allowEmptyList'    => false,
        'enableGuessTitle'  => true,
        'min'               => 1, // should be at least 2 rows
        #'addButtonPosition' => MultipleInput::POS_HEADER, // show add button in the header
        'addButtonOptions' => [
            'class' => 'btn-xs btn-success',
            #'label' => 'add' // also you can use html code
        ],
        'removeButtonOptions' => [
            'class' => 'btn-xs btn-danger',
            #'label' => 'remove'
        ],
        'columns' => [
            [
                'name' => 'currency',
                'title' => 'Currency',
                'type'=> MaskedInput::className(),
                'options' => [
                    'clientOptions' => [
                        'alias' =>  'currency',
                        'prefix' => '',
                        'radixPoint'=> ",",
                        'groupSeparator' => '.',
                        'step' => 1,
                        'autoGroup' => true
                    ],
                    'options' => [
                        'class'=>'form-control',
                    ],
                ],
            ],
        ],
    ]);
    ?>
        
	<?php if (!Yii::$app->request->isAjax){ ?>
	  	<div class="form-group">
	        <?= Html::submitButton($model->isNewRecord ? Yii::t('app', 'Create') : Yii::t('app', 'Update'), ['class' => $model->isNewRecord ? 'btn btn-success' : 'btn btn-primary']) ?>
	    </div>
	<?php } ?>

    <?php ActiveForm::end(); ?>
    
</div>
```

controller
```
    public function actionCreateTest()
    {
        $request = Yii::$app->request;
        $model = new Model();

        if($request->isAjax){
            /*
            *   Process for ajax request
            */
            Yii::$app->response->format = Response::FORMAT_JSON;
            if($request->isGet){
                return [
                    'content'=>$this->renderAjax('create', [
                        'model' => $model,
                    ]),
                    'footer'=> Html::button(Yii::t('app', 'Close'),['class'=>'btn btn-default pull-left','data-dismiss'=>"modal"]).
                        Html::button(Yii::t('app', 'Save'),['class'=>'btn btn-primary','type'=>"submit"])
                ];
            }else
                if($model->load($request->post()) && $model->save()){
                return [
                    'forceReload'=>'#crud-datatable-pjax',
                    'content'=>'<span class="text-success">Success</span>',
                    'footer'=> Html::button(Yii::t('app', 'Close'),['class'=>'btn btn-default pull-left','data-dismiss'=>"modal"]).
                        Html::a(Yii::t('app', 'Create More'),['create'],['class'=>'btn btn-primary','role'=>'modal-remote'])
                ];
            }else{
                return [
                    'content'=>$this->renderAjax('create', [
                        'model' => $model,
                    ]),
                    'footer'=> Html::button(Yii::t('app', 'Close'),['class'=>'btn btn-default pull-left','data-dismiss'=>"modal"]).
                        Html::button(Yii::t('app', 'Save'),['class'=>'btn btn-primary','type'=>"submit"])
                ];
            }
        }else{
            /*
            *   Process for non-ajax request
            */
            if ($model->load($request->post()) && $model->save()) {
                return $this->redirect(['view', 'id' => $model->id]);
            } else {
                return $this->render('create', [
                    'model' => $model,
                ]);
            }
        }
    }
```

this is with 1.3.1 version of multipleinput
working on dialog when i use ajax, and same error if don't use dialog

also i changed for new version like you write in update instruction and then don't work in dialog

**Comment by andjelko** [comment]
index
```
use yii\helpers\Html;
use yii\bootstrap\Modal;
use johnitvn\ajaxcrud\CrudAsset; 

CrudAsset::register($this);

<?= Html::a('<i class="glyphicon glyphicon-plus"></i>', ['create-test'], ['role'=>'modal-remote','title'=> 'Create new ','class'=>'btn btn-default']) ?>


<?php Modal::begin([
    "id"=>"ajaxCrudModal",
    "size" => "modal-wide",
    "footer"=>"",// always need it for jquery plugin
])?>
<?php Modal::end(); ?>

```

**Comment by andjelko** [comment]
Did you find the bug?

**Comment by unclead** [comment]
sorry i don't have free time now to handle it

**Comment by unclead** [comment]
i've tried to reproduce a bug using your code but i didn't understand how to compose your code to make working example.

if this issue is still actual please put all your code into gist and give me a link

**Comment by unclead** [comment]
it seems i know where the bug is but i can't fix it now because of lack free time
you can use 1.3.1 until i fix it

**Comment by unclead** [comment]
fixed via e4e94df24c902f5f33b3a0edde222b28063dcc24

i have to update to 2.7.0
